.github/workflows/cd-deploy-nodes-gcp.yml

```
@@ -118,99 +98,37 @@ jobs:
 
   # Test that Zebra works using the default config with the latest Zebra version.
   test-configuration-file:
-    name: Test Zebra CD Docker config file
-    timeout-minutes: 15
-    runs-on: ubuntu-latest
+    name: Test Zebra default Docker config file
```

Optional: we can either change the job name or change the branch protection rules to use the new name

```suggestion
    name: Test Zebra CD Docker config file
```